src/Point.cpp

```diff
@@ -33,7 +33,7 @@
 using namespace std;
 using namespace openshot;
 
-// Default constructor (defaults to 0,0)
+// Default constructor (defaults to 1,0)
```

Comment:
Same thing here. Look down 3 lines, and the fact that it defaults to `(1,0)` is completely apparent from the `Coordinate(1, 0)` in the code itself.